shared/test/test_net_sim_api.rb

```diff
@@ -1,6 +1,14 @@
+# Unit tests for NetSimApi
+# This uses a fake Redis service by default, but can be configured to use a
+# real instance on localhost by setting the USE_REAL_REDIS environment variable;
+# e.g. "USE_REAL_REDIS=true ruby test_redis_property_bag.rb".
+# Caution: This test is destructive, clears the whole Redis instance between
+# tests, so be careful when using real redis.
```

Comment:
I'm putting in a check, but it should be a non-issue because even when running against 'real Redis' these tests always look for Redis on `localhost`.  On production there is no Redis server running on localhost - production Redis is an Elasticache node.